.circleci/config.yml

```diff
@@ -7,7 +7,6 @@ jobs:
         BUNDLE_JOBS: 3
         BUNDLE_RETRY: 3
         BUNDLE_PATH: vendor/bundle
-        MYSQL_DATABASE: circle_test
```

Comment:
i ended up just passing the default sul_pub test DB name to the circle mysql docker image config, instead of passing the DB name into the app and having the app config pick it up.